packages/vscode-dependencies-validation/test/vscodeMocks.ts

```diff
@@ -0,0 +1,189 @@
+import { NPM_DEPENDENCY_ISSUES_CODE } from "../src/constants";
```

Comment:
its not a huge deal, but needing to implement these VSCode APIs makes me wonder
if we should (later) make an attempt to use VSCode-Test or https://github.com/SAP/vscode-test-cli
and get those implementations for free as the **unit** tests would be running in an env that has VSCode

Reply:
one disadvantage of this approach is that we may end up with placeholder classes we no longer need
when the source code / tests gets refactored.

we need to discuss these two approaches online next week...